internal/pkg/cli/task_run.go

```diff
@@ -909,6 +928,9 @@ Run a task with a command.
 			if cmd.Flags().Changed(dockerFileFlag) {
 				opts.isDockerfileSet = true
 			}
+			if cmd.Flags().Changed(dockerFileContextFlag) {
+				opts.isDockerfileContextSet = true
+			}
```

Comment:
We probably don't need this as we dont' set the default value in https://github.com/aws/copilot-cli/pull/3046/files#diff-eec95b70c382971e4ec97c1db10feebe2b837deaba537fc9ec999e43ea62a9bbR946. We could simply check if it equals to "" to determine if this flag is set by users or not.

Reply:
👍 Done!